config/test.yml.erb

```diff
@@ -68,3 +68,9 @@ use_cookie_dcdo: true
 
 # Allows Geolocation to be altered via cookies.
 use_geolocation_override: true
+
+# Number of workers to start when running `rake build:start_active_job_workers`:
+<% if test_system? && chef_managed %>
+active_job_backend_n_workers_to_start: 10
+active_job_backend_rolling_restart_in_n_batches: 2
+<% end -%>
```

Comment:
how about reducing the number of workers back down to 1 for drone, due to the memory constraints in that environment?

Reply:
per slack discussion: workers fork from an image that has rails preloaded, and two workers will share the majority of RAM. I'd like to keep CI as similar to prod as possible, and having jobs be able to run in parallel means certain concurrency issues could show up in Drone that otherwise would be masked.